Library/ViewModels/SettingsViewModel.swift

```diff
@@ -541,9 +541,9 @@ SettingsViewModelOutputs {
     self.postLikesTappedProperty.value = selected
   }
 
-  fileprivate let showPublicProfileProperty = MutableProperty(false)
+  fileprivate let privateProfileEnabledProperty = MutableProperty(false)
```

Comment:
Since this has flipped, is the default still `false`?

Reply:
Duh yes you're right, it should be `true`. Thanks!